docs/sources/flow/get-started/install/kubernetes.md

```diff
@@ -20,20 +20,34 @@ weight: 200
 
 # Deploy {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-{{< param "PRODUCT_NAME" >}} can be deployed on Kubernetes by using the Helm chart for {{< param "PRODUCT_ROOT_NAME" >}}.
+You can deploy {{< param "PRODUCT_NAME" >}} on Kubernetes using our Helm chart.
 
-## Before you begin
+## Dedicated guides
+
+We recommend using our dedicated guides for each telemetry signal to deploy a 
+production-ready {{< param "PRODUCT_NAME" >}} on Kubernetes:
+
+* For metrics, follow our dedicated [collect Prometheus metrics on Kubernetes][collect-prometheus] guide.
+* For logs, follow our dedicated [collect logs on Kubernetes][collect-logs] guide.
+* For anything else, follow the [generic Helm installation](#generic-helm-installation) instructions below.
+
+[collect-prometheus]: {{< relref "../../tasks/kubernetes/collect-prometheus.md" >}}
+[collect-logs]: {{< relref "../../tasks/kubernetes/collect-logs.md" >}}
+
+If you want to collect multiple types of telemetry, we recommend deploying separate  
+{{< param "PRODUCT_NAME" >}} workloads for each telemetry type.
+
+## Generic Helm installation
```

Comment:
I'm not sure that I follow why we are bumping all these headings up a level. If this is just/only deploy on K8s using Generic Helm, then we should state that in the top H1 level. Or will this topic have other info about deploying on K8s with methods other than Helm?

Reply:
The reason is that I intended to have a hierarchy here:

- Deploy Agent on Kubernetes
  - Deploy using the dedicated guides
    - We provide a list of links to pages that describe how to do that for each telemetry signal
  - Deploy using a generic Helm installation
    - The instructions for this are included on this page. They are meant for those that didn't find a dedicated guide for their use case in the list above.

Do you have any recommendations how to handle this better? 
We need the dedicated guides to provide sufficient level of instruction. But we also need a generic Helm installation guide for those that didn't find a matching dedicated guide and are willing to dive a bit deeper themselves.

Reply:
If it's for future layout adjustments, then lets do it :-) Docs are fluid and adjustable.